packaging/nmstate.spec

```diff
@@ -41,10 +41,8 @@ Requires:       python3-nispor
 Summary:        nmstate plugin for OVS database manipulation
 Requires:       python3-%{libname} = %{?epoch:%{epoch}:}%{version}-%{release}
 %if 0%{?rhel}
-# The python-openvswitch rpm package is not in the same repo with nmstate,
-# require only if openvswitch is installed.
+# The python-openvswitch rpm package required only if openvswitch is installed.
 Requires:       (python3dist(ovs) if openvswitch)
-Recommends:     python3dist(ovs)
```

Comment:
Please do not remove this. If openvswitch is not present we need to recommend the installation.